idaes/apps/uncertainty_propagation/examples/uncertainty_propagation_rooney.py

```diff
@@ -0,0 +1,35 @@
+##############################################################################
```

Comment:
Do we need 3 version of this example? Does it make sense to consolidate into one version?

Reply:
I think one version is enough. I made them for just personal testing.

Reply:
Please remove any extra version of this if you have not already. Then resolve this conversation.

Reply:
@JanghoPark-LBL Is this done? If so, just click "resolve conversation".